### PR TITLE
Bugfix/nh wv mi

### PR DESCRIPTION
**Pull request by Tommi-Tsuruga**
**Addresses issue(s): **

## What this does

This just does small bugfixes for the first of the month files

### Side effects

n/a

## How to test

1. Tested already with batch jobs ran off my sweet new image

## Checklist

- [ ] This has been tested locally.
  - Notes: 
- [ ] [Commented code](https://docs.google.com/document/d/1M_i2i3V2aNq6Yiofwj5Su8mKGLs1ooQOJGHR-37WZzs/edit) in a way that is useful for others.
  - Notes: 
- [ ] Updated or created relevant documentation.
  - Notes: 
- [ ] Added automated tests relevant to this new code.
  - Notes: 

## Other context

- Requires dependencies update: **NO**
- This is directly related to a pull request in another repo? **YES**
  - Will need an inspector pr after this one to update the reggie pointer

### CODE REVIEW

File: reggie/ingestion/preprocessor/michigan_preprocessor.py [hunk 1]
```diff
@@ -188,6 +187,11 @@ def fill_empty_columns(df):
                     },
                     inplace=True,
                 )
+            # This is new
+            elif ("IS_ABSENTEE_VOTER" not in hdf.columns) and (
+                "IS_PERMANENT_ABSENTEE_VOTER" not in hdf.columns
```
Comment (zzolo) [inline reggie/ingestion/preprocessor/michigan_preprocessor.py:192]: (minor comments) Why are we checking both columns here?

**Review by zzolo — APPROVED** [review]
Minor comment.  Also, I don't see a WV change, though you had described that there was something needed for it?

File: reggie/ingestion/preprocessor/michigan_preprocessor.py (hunk 1 above)
Comment (Tommi-Tsuruga) [reply reggie/ingestion/preprocessor/michigan_preprocessor.py:192]: It's the opposite, we're checking that neither of those columns is there. I suppose, we could just check if the renamed column is not there at this point but we don't want to overwrite the column that exists, if it does with nans

Comment (Tommi-Tsuruga) [reply reggie/ingestion/preprocessor/michigan_preprocessor.py:192]: Oh also WV just needed a reupload, the zip was corrupted.

**Pull request merged**